### DISCUSSION TITLE
S3 Integration

### DISCUSSION THREAD

**Discussion by cubismod** [Isawan/terrashine#224] [Q&A] [answered]
Heyo! Very cool project. I was taking a look through the docs and noticed that [the introduction page](https://isawan.github.io/terrashine/introduction.html) mentioned using S3 object storage as a backend cache. How would you configure terrashine to use S3 as a caching mechanism? I don't see that mentioned on the other docs pages.

**Comment by Isawan** [comment, marked as answer]
Hi, I've updated the documentation to include more information on setting up with S3 and generally improved the documentation in general. Could you let me know if that covers what you need? https://isawan.github.io/terrashine/s3-compatible-object-storage.html

**Reply by cubismod** [reply]
Thank you, yes that covers what I need!
